src/inc/corinfo.h

```diff
@@ -2129,6 +2129,12 @@ class ICorStaticInfo
             CORINFO_CLASS_HANDLE        classHnd
             ) { return false; }
 
+    // Is the given type marked as [intrinsic]?
+    // This defaults to false.
+    virtual bool isIntrinsicType(
+            CORINFO_CLASS_HANDLE        classHnd
+            ) { return false; }
```

Comment:
This should be `= 0` to match other methods. I think `isInSIMDModule` is outlier - not sure why it is missing `= 0`.

Reply:
Good catch, will fix.